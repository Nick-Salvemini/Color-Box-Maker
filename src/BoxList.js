import React from "react";
import './BoxList.css';