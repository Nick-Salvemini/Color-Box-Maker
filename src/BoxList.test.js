import React from "react";
import { render } from "@testing-library/react";
import BoxList from "./BoxList";

describe('BoxList component', () => {
    it('renders without crashing', () => {
        render(<BoxList />);
    });

    it('matches snapshot', () => {
        const { container } = render(<BoxList />);
        expect(container).toMatchSnapshot();
    });
});