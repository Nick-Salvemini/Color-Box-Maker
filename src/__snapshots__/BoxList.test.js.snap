// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxList component matches snapshot 1`] = `
<div>
  <h1>
    Color Box Maker
  </h1>
  <form>
    <label
      for="color"
    >
      Color
    </label>
    <input
      id="color"
      name="color"
      placeholder="CSS-friendly Color"
      type="text"
      value=""
    />
    <label
      for="height"
    >
      Height
    </label>
    <input
      id="height"
      name="height"
      placeholder="Height in Pixels"
      type="text"
      value=""
    />
    <label
      for="width"
    >
      Width
    </label>
    <input
      id="width"
      name="width"
      placeholder="Width in Pixels"
      type="text"
      value=""
    />
    <button>
      Add Box
    </button>
  </form>
  <div />
</div>
`;
